src/FIOSDK.ts

```diff
@@ -439,6 +440,29 @@ export class FIOSDK {
     return addPublicAddress.execute(this.privateKey, this.publicKey)
   }
 
+  /**
+   * This call allows a any number of public addresses matching the blockchain code, the token code and the public address to be removed from the FIO Address.
+   *
+   * @param fioAddress FIO Address which will be mapped to public address.
+   * @param publicAddresses a list of publicAddresses, each containing chain_code, token_code, and public_address.
+   * @param maxFee Maximum amount of SUFs the user is willing to pay for fee. Should be preceded by /get_fee for correct value.
+   * @param technologyProviderId FIO Address of the wallet which generates this transaction.
+   */
+  public removePublicAddress(
```

Comment:
should it be `removePublicAddresses` since we pass an array there?

Reply:
yes it should, ill change it...